pkg/collect/host_subnetavailable_test.go

```diff
@@ -0,0 +1,236 @@
+package collect
+
+import (
+	"net"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+/*
```

Comment:
You could uncomment this so as to test some invalid configuration input use cases

Reply:
yea I'm thinking about if I have the cycles to get this part done... :) I'll see how I go. would be nice to have that full test all the way from start to finish. I've at least got the logic under the hood being exercised independently (as unit tests) already

Reply:
I thought more about this test, but it's a bit of a PITA to add it as it won't run reliably on anything except a Linux machine (due to the dependency on `/proc/net/route`). there's already unit tests on the underlying functionality beneath the collector itself, I'm comfortable with that for now...